cardshifter-server/src/test/java/com/cardshifter/server/main/MainServerTest.java

```diff
@@ -0,0 +1,27 @@
+package com.cardshifter.server.main;
+
+import com.cardshifter.server.model.MainServer;
+import com.cardshifter.server.model.Server;
+import org.apache.log4j.PropertyConfigurator;
+import org.junit.Test;
+
+/**
+ * Created by marc on 2017-02-07.
+ */
+public class MainServerTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void testThrowWhenNoModAreLoaded() {
```

Comment:
Must be because you have mods when you run tests. I should amend the test to make sure there is no mods in the path, but if you pass a folder when you run the tests it may fail the test. Will try to come up with a solution.